docs/user-guide/cli-arguments.md

```diff
@@ -22,6 +22,7 @@ They are set in the container spec of the `ingress-nginx-controller` Deployment
 | `--disable-sync-events` | Disables the creation of 'Sync' Event resources, but still logs them |
 | `--dynamic-configuration-retries` | Number of times to retry failed dynamic configuration before failing to sync an ingress. (default 15) |
 | `--election-id`                    | Election id to use for Ingress status updates. (default "ingress-controller-leader") |
+| `--leader-election-ttl-in-seconds`          | Leader election time to live in seconds |
```

Comment:
Can we please call it just `election-ttl`? Right now there already is an `election-id`, so those would be aligned then. Apart form that I would not include the unit in the argument name and rather handle durations (which can be everything time unit).